velox/functions/lib/string/StringImpl.h

```diff
@@ -122,6 +122,25 @@ FOLLY_ALWAYS_INLINE int64_t cappedLength(const T& input, size_t maxLength) {
   }
 }
 
+/// Return a capped bytes(controlled by maxCharacters) of a string.
```

Comment:
nit: a capped length in bytes (...)

and modify the comment on L114: a capped length in characters (...)